fastestimator/op/numpyop/univariate/__init__.py

```diff
@@ -12,6 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 # ==============================================================================
+# see a complete list of effect in albumentation page: https://albumentations-demo.herokuapp.com
```

Comment:
effects plural. also, this notice should probably go in NumpyOp since the page also includes multi-variate augmentations ('professional mode')

Reply:
added this notice in multivariate init as well.